static/app/components/events/interfaces/spans/traceView.spec.tsx

```diff
@@ -325,6 +325,163 @@ describe('TraceView', () => {
+        'a000000000000000',
+        event
+      );
+      const waterfallModel = new WaterfallModel(event);
+
+      const eventsTraceMock = MockApiClient.addMockResponse({
+        url: `/organizations/${organization.slug}/events-trace/${event.contexts.trace?.trace_id}/`,
+        method: 'GET',
+        statusCode: 200,
+        body: [
+          event,
+          {
+            errors: [],
+            event_id: '998d7e2c304c45729545e4434e2967cb',
+            generation: 1,
+            parent_event_id: '2b658a829a21496b87fd1f14a61abf65',
+            parent_span_id: 'b000000000000000',
+            project_id: project.id,
+            project_slug: project.slug,
+            span_id: '8596e2795f88471d',
+            transaction:
+              '/api/0/organizations/{organization_slug}/events/{project_slug}:{event_id}/',
+            'transaction.duration': 159,
+            'transaction.op': 'http.server',
+          },
+          {
+            errors: [],
+            event_id: '59e1fe369528499b87dab7221ce6b8a9',
+            generation: 1,
+            parent_event_id: '2b658a829a21496b87fd1f14a61abf65',
+            parent_span_id: 'b000000000000000',
+            project_id: project.id,
+            project_slug: project.slug,
+            span_id: 'aa5abb302ad5b9e1',
+            transaction:
+              '/api/0/organizations/{organization_slug}/events/{project_slug}:{event_id}/',
+            'transaction.duration': 159,
+            'transaction.op': 'middleware.nextjs',
+          },
+        ],
+      });
+
+      const eventsTraceLightMock = MockApiClient.addMockResponse({
+        url: `/organizations/${organization.slug}/events-trace-light/${event.contexts.trace?.trace_id}/`,
+        method: 'GET',
+        statusCode: 200,
+        body: [
+          event,
+          {
+            errors: [],
+            event_id: '998d7e2c304c45729545e4434e2967cb',
+            generation: 1,
+            parent_event_id: '2b658a829a21496b87fd1f14a61abf65',
+            parent_span_id: 'b000000000000000',
+            project_id: project.id,
+            project_slug: project.slug,
+            span_id: '8596e2795f88471d',
+            transaction:
+              '/api/0/organizations/{organization_slug}/events/{project_slug}:{event_id}/',
+            'transaction.duration': 159,
+            'transaction.op': 'http.server',
+          },
+          {
+            errors: [],
+            event_id: '59e1fe369528499b87dab7221ce6b8a9',
+            generation: 1,
+            parent_event_id: '2b658a829a21496b87fd1f14a61abf65',
+            parent_span_id: 'b000000000000000',
+            project_id: project.id,
+            project_slug: project.slug,
+            span_id: 'aa5abb302ad5b9e1',
+            transaction:
+              '/api/0/organizations/{organization_slug}/events/{project_slug}:{event_id}/',
+            'transaction.duration': 159,
+            'transaction.op': 'middleware.nextjs',
+          },
```

Comment:
These look the same as the mocks right above, right? If yes, would be nice to re-use!

Reply:
It's almost the same, the difference is that it has two embedded transactions set under the same parent as opposed to one. But you bring up a good point, I should make a const and then copy + modify it for this test to make that more clear